### DISCUSSION TITLE
Select R interpreter via path

### DISCUSSION THREAD

**Discussion by joshualeond** [posit-dev/positron#3895] [Q&A] [answered]
Is it possible to enter the path for your R interpreter directly? 

I'm attempting to use the Pixi package manager like referenced in https://github.com/posit-dev/positron/issues/3724. For Python we can enter the path directly but this doesn't seem to work for R.

![image](https://github.com/posit-dev/positron/assets/3802502/9caedd58-9b16-4f48-8e91-cf9b51b254ee)

Thanks!

**Comment by jennybc** [comment, marked as answer]
No, this is not currently possible but the need is already being tracked in #2235.

**Reply by joshualeond** [reply]
Thank you!
